android/sdk/src/main/java/com/taobao/weex/utils/WXUtils.java

```diff
@@ -569,6 +572,25 @@ public static int getNumberInt(Object value, int defaultValue){
     }catch (Exception e){return  defaultValue;}
   }
 
+  public static boolean checkGreyConfig(String group,String key,String defaultValue){
+      Log.e("test->", "enter checkGreyConfig");
```

Comment:
Delete it or change to `WXLogUtils.i`
* Line 576
* Line 579
* Line 590

Reply:
done